pnpm-lock.yaml

```diff
@@ -570,7 +598,7 @@ importers:
         version: 3.666.0(@aws-sdk/client-sts@3.666.0)
       '@getzep/zep-cloud':
         specifier: 1.0.12
-        version: 1.0.12(@langchain/core@0.3.30(openai@4.78.1(encoding@0.1.13)(zod@3.24.1)))(encoding@0.1.13)(langchain@0.3.11(fd386e1130022c8548c06dd951c5cbf0))
+        version: 1.0.12(@langchain/core@0.3.30(openai@4.78.1(encoding@0.1.13)(zod@3.24.1)))(encoding@0.1.13)(langchain@0.3.11(d1e86e144e3517fab3dbb7a92ab7f45a))
```

Comment:
there seem to be quite a few changes here that shouldn't be.
I ran an update on this branch locally, and many of these were reverted.

I'm not sure why this keeps happening to so many of us, but we should look into if this is a bug in pnpm, or in our dev setup.

Reply:
Ran this and it's now much smaller:

```
rm -rf node_modules packages/**/node_modules cypress/node_modules && gco origin/master pnpm-lock.yaml && pnpm i
```